### ISSUE TITLE
Ground truth errors bias weaker scores across all models

### ISSUE THREAD

**Issue by meanaverage** [Tencent/DiffSpot#2]
While using this to enhance a mutimodal pipeline, we've noticed that while the test corpus remains reliable for benchmarking due to the frozen state of the test examples, it should probably not be used as unreviewed training data as far as an actual pass/fail for vision models and a true pass/fail detection.

<img width="1300" height="877" alt="Image" src="https://github.com/user-attachments/assets/942a64be-01c1-4dcf-9068-9f0b66915d2d" />

<img width="1683" height="1128" alt="Image" src="https://github.com/user-attachments/assets/92550295-250a-491f-b4e2-4895df5e1f85" />

A simple spot-review of examples which our own harness plus additional context could not pass, on review, shows the defect is often with the sample and/or the ground truth being inaccurate.

Examples included:
1c73243f-9814-4252-a6b5-af03062f39c4
f69fb188-7bbc-4b17-80b0-177dd2b6bf18

First, and only two, that I've checked out of thousands, but you can see the problem with this method. This paints frontier models and the other models tested in a potentially weaker light as to their true abilities.

From the paper: 
"The majority of reviewers agreed with the automated judge 97.1% of the time. All three reviewers unanimously considered the judge wrong in 5/477 = 1.0% of cases"

More helpful study information would be the expected error rate of the ground truths. I could not find that metric or that any review was done.
